manifest/com.scicortex/CyberFingerMod/info.json

```diff
@@ -30,6 +30,13 @@
 				"url": "https://github.com/DrSciCortex/CyberFingerMod/releases/download/v1.1.1/CyberFingerMod.dll",
 				"sha256": "98e75279f96410ff721375761066845e8b5ca1c4c1d186b91374a259f7b64a14"
 			}]
+		},
+		"1.1.2": {
+			"releaseUrl": "https://github.com/DrSciCortex/CyberFingerMod/releases/tag/v1.1.2",
+			"artifacts": [{
+				"url": "https://github.com/DrSciCortex/CyberFingerMod/releases/download/1.1.2/CyberFingerMod.dll",
```

Comment:
Non-blocking: built in debug

Reply:
btw Elements.Core.MathX contains a lot of math functions if you want to use ones provided there instead of having a copy within your code. Such as Clamp01 or normal Clamp